lib/analog_clock.dart

```diff
@@ -29,6 +30,7 @@ class AnalogClock extends StatefulWidget {
       this.showNumbers = true,
       this.showSecondHand = true,
       this.showAllNumbers = false,
+      this.useMilitaryTime = false,
```

Comment:
To be able to backward compatible, I think `useMilitaryTime` should be true by default.

Reply:
I have submitted the change to make it backwards compatible.